networkd/network_server.ml

```diff
@@ -859,6 +859,31 @@ module Bridge = struct
 		) ()
 end
 
+module PVS_proxy = struct
+	open PVS_proxy
+
+	let path = ref "/var/run/pvsproxy"
+	
+	let do_call call =
+		try
+			Jsonrpc_client.with_rpc ~path:!path ~call
+		with e ->
+			error "Error when calling PVS proxy: %s" (Printexc.to_string e);
+			raise PVS_proxy_connection_error
+
+	let configure_farm _ dbg config =
+		debug "Configuring PVS proxy for farm %s" config.farm_uuid;
+		let call = {Rpc.name = "configure_farm"; params = [rpc_of_t config]} in
+		let _ = do_call call in
```

Comment:
Yes, but at this point, we don't know which errors the daemon may return yet. We will have to fill this in later.